CHANGES.txt

```diff
@@ -43,6 +43,8 @@ Changes
 Fixes
 =====
 
+ - Fixed an issue with loading 'path.logs' from the Crate config file.
```

Comment:
I would say:
"Fixed an issue that caused `path.logs` setting in `crate.yml` to be ignored."